docs/agent-api.asciidoc

```diff
@@ -787,3 +787,34 @@ This enables log correlation to APM traces with text-only loggers.
 "trace.id=abc123 span.id=abc123"
 ----
 // end::currentTraceIds[]
+
+[[apm-register-custom-metrics]]
+==== `apm.registerMetric(name, callback)`
```

Comment:
```suggestion
```

Reply:
Argh, sorry - GitHub didn't include this suggestion. Need to get rid of this extra title for the docs to build.